src/Billing/Utilities/StripeWebhookUtility.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Bit.Billing.Utilities;
+
+public static class StripeWebhookUtility
```

Comment:
I assume this is a duplicate of the the file here? `src/Billing/Services/Implementations/WebhookUtility.cs`